finetune/nn/target_blocks.py

```diff
@@ -290,11 +290,15 @@ def sequence_labeler(hidden, targets, n_targets, config, pad_id, multilabel=Fals
     """
     with tf.variable_scope('sequence-labeler', reuse=reuse):
         nx = config.n_embed
+
         def seq_lab_internal(hidden):
-            attn_fn = functools.partial(attn, scope="seq_label_attn", n_state=nx, n_head=config.seq_num_heads,
-                                            resid_pdrop=config.resid_p_drop, attn_pdrop=config.attn_p_drop,
-                                            train=train, scale=False, mask=False)
-            n = norm(attn_fn(hidden) + hidden, 'seq_label_residual')
+            if config.base_model.is_bidirectional:
```

Comment:
It might be worth modifying the docstring slightly to also apply to BERT. I'm specifically thinking of the line `The representations fed into this model are necessarily future masked because a language modelling loss is the original objective of the featurizer.`